lib/FunctionModuleTemplatePlugin.js

```diff
@@ -10,8 +10,8 @@ class FunctionModuleTemplatePlugin {
 	apply(moduleTemplate) {
 		moduleTemplate.plugin("render", function(moduleSource, module) {
 			const source = new ConcatSource();
-			const defaultArguments = [module.moduleArgument || "module", module.exportsArgument || "exports"];
-			if((module.arguments && module.arguments.length !== 0) || module.hasDependencies(d => d.requireWebpackRequire !== false)) {
```

Comment:
These two properties are not used in the webpack codebase itself, but may be used by plugins.

Don't remove them.